SimTracker/TrackerMaterialAnalysis/test/trackingMaterialProducer10GeVNeutrino_ForPhaseII.py

```diff
@@ -1,24 +1,20 @@
 #!/usr/bin/env cmsRun
 
+# cmsRun trackingMaterialProducer.py nEvents=1000
+
 import FWCore.ParameterSet.Config as cms
+from FWCore.ParameterSet.VarParsing import VarParsing
 
 process = cms.Process("Geometry")
 
 readGeometryFromDB = False
 
-# N.B. for the time being we load the geometry from local
-# XML, whle in future we will have to use the DB. This is
-# only a temporary hack, since the material description has
-# been updated in release via XML and the DB is behind.
-if not readGeometryFromDB:
-  process.load('Configuration.Geometry.GeometryExtended2026D41_cff')
-else:
-# GlobalTag and geometry via GT
-  process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
-  from Configuration.AlCa.GlobalTag_condDBv2 import GlobalTag
-  process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:phase2_realistic', '')
+process.load('Configuration.Geometry.GeometryExtended2026D49_cff')
 
 process.load('FWCore.MessageService.MessageLogger_cfi')
+process.MessageLogger.destinations.extend(["debugTrackingMaterialProducer"])
+process.MessageLogger.categories.append("TrackingMaterialProducer")
```

Comment:
is the effect of this completely equivalent to the one of the removed `customizeMessageLogger` function?

Reply:
The function was not being executed https://github.com/cms-sw/cmssw/blob/c53ec47424bd5e5c7fb2eafb31c06cb579d0287d/SimTracker/TrackerMaterialAnalysis/test/trackingMaterialAnalyser_ForPhaseII.py#L68. This should do the work needed